charts/intel-gpu-resource-driver/README.md

```diff
@@ -34,10 +35,12 @@ helm show values intel/intel-gpu-resource-driver
 
 You may also run `helm show values` on this chart's dependencies for additional options.
 
-|parameter| value |
-|---------|-----------|
-| `image.repository` | `intel` |
-| `image.tag` | `v0.3.0` |
+| Key | Type | Default |
+|-----|------|---------|
+| image.repository | string | `intel` |
+| image.name | string | `"IfNotPresent"` |
+| image.pullPolicy | string | `"intel-gpu-resource-driver"` |
+| image.tag | string | `""` |
```

Comment:
Do not seem to be valid image name, pull policy or tag.

Reply:
should I put there the tag v0.4.0 or is it ok to leave empty string?

Reply:
Default **is** v0.4.0, isn't it?

Reply:
yes, I will update